mds_asssignment_1/guestbook/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils import timezone
+
+class Entry(models.Model):
+    fname = models.CharField(max_length=50, verbose_name='First Name')
+    lname = models.CharField(max_length=50, verbose_name='Last Name', blank=True)
+    email = models.CharField(max_length=100, blank=True)
+    comment = models.CharField(max_length=500)
+    date = models.DateTimeField(default=timezone.now)
```

Comment:
Very nice understanding of how this `default` attribute works here. There's `auto_now_add` as well which does same thing as you are here.

You could add `editable=False` to remove this field from the admin (you can still force it to be displayed).